stl/inc/istream

```diff
@@ -359,8 +359,8 @@ public:
         return get(_Str, _Count, _Myios::widen('\n'));
     }
 
-    basic_istream& __CLR_OR_THIS_CALL get(
-        _Elem* _Str, streamsize _Count, _Elem _Delim) { // get up to _Count characters into NTCS, stop before _Delim
+    basic_istream& __CLR_OR_THIS_CALL get(_Elem* _Str, streamsize _Count, _Elem _Delim) {
+        // get up to _Count characters into NTCS, stop before _Delim

```

Comment:
No change requested but I do want to raise a concern.

I see that @CaseyCarter asked about comment wrapping in https://github.com/microsoft/STL/pull/2033#discussion_r691414753 . I'm not inherently opposed to drive-by cleanups, and I certainly have a personal tendency to lump too many changes into a single PR. However, I want to note that this does make it somewhat more difficult to review a PR. Ordinarily I probably wouldn't mention it, but iostreams behavioral changes are some of the riskiest possible changes in the entire STL, so I think we should be extra-cautious about PR cleanliness with them.

These detached comments aren't really mixed with the behavioral changes so it's fine to leave them in this PR.